daemon/src/payout_curve.rs

```diff
@@ -153,6 +153,8 @@ pub enum Error {
     #[error("matrix must be square")]
     MatrixMustBeSquare,
     #[error("evaluation outside parametric domain")]
+    SingularMatrix,
+    #[error("cannot invert singular matrix")]
```

Comment:
I think some lines got flipped here, looks like the `#[error("cannot invert singular matrix")]` should be above `SingularMatrix` and `#[error("evaluation outside parametric domain")]` on top of `InvalidDomain` ?